tasks/sign-index-image/tests/test-sign-index-image.yaml

```diff
@@ -0,0 +1,105 @@
+---
+apiVersion: tekton.dev/v1
+kind: Pipeline
+metadata:
+  name: test-sign-index-image
+spec:
+  description: Test creating a internal request to sign an image
+  workspaces:
+    - name: tests-workspace
+  tasks:
+    - name: setup
+      workspaces:
+        - name: data
+          workspace: tests-workspace
+      taskSpec:
+        workspaces:
+          - name: data
+        steps:
+          - name: setup-values
+            image: quay.io/hacbs-release/release-utils:85ab98a7ec63c3d8d9ec3c4982ff0e581bcb3c83
+            script: |
+              #!/usr/bin/env sh
+              set -eux
+
+              cat > $(workspaces.data.path)/data.json << EOF
+              {
+                "sign": {
+                  "request": "hacbs-signing-pipeline",
+                  "configMapName": "signing-config-map"
+                }
+              }
+              EOF
+    - name: run-task
+      taskRef:
+        name: sign-index-image
+      params:
+        - name: requester
+          value: testuser
+        - name: referenceImage
+          value: quay.io/testrepo/testimage:tag
+        - name: manifestDigestImage
+          value: quay.io/testrepo/testimage@sha256:0000
+      workspaces:
+        - name: data
+          workspace: tests-workspace
+      runAfter:
+        - setup
+    - name: check-result
+      workspaces:
+        - name: data
+          workspace: tests-workspace
+      taskSpec:
+        steps:
+          - name: check-result
+            image: quay.io/hacbs-release/release-utils:a9603c0ae1982be5e1142d44765b1475adc7ad40
+            script: |
+              #!/usr/bin/env sh
+              set -eux
+
+              # First internal request with fixed tag
```

Comment:
It seems this is a leftover from duplicating another test.

Reply:
indeed. Removed.